### ISSUE TITLE
Annotation Overlay

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#1185]
Hi @constantinpape,

I am working on a new functionality to overlay any table column as text on the image:

<img width="1165" alt="image" src="https://github.com/user-attachments/assets/5bd507c7-e18f-497a-9ffb-900558a0f559">

Doing so it looks like as if the anchor points in the Platy cell segmentation tend to be at the border of the segments. Is that true?

**Comment by constantinpape** [comment]
> Doing so it looks like as if the anchor points in the Platy cell segmentation tend to be at the border of the segments. Is that true?

No, they should be in the center. But looking at this in 2D may give a bit misleading impression, since this is the centroid point in 3D.

**Comment by tischi** [comment]
Do you mean **medoid**? Because the **centroid** can be outside the shape, which is not allowed by the spec for the "anchor points:...

**Comment by constantinpape** [comment]
No, it's the centroid, but gets moved inside of the object if it is outside of it.
